devices.js

```diff
@@ -2302,6 +2302,13 @@ const devices = [
             await configureReporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['ZB-RGBCW'],
+        model: 'Zigbee 3, RGW+W+C',
```

Comment:
Please fill in a meaningful model here, preferably something that can be Googled and leads to this bulb.

Reply:
Unfortunately, there is no meaningul model in the paper manual nor in the Aliexpress lot. Just "ZigBee 3.0 Tuya Smart Led Lamp Bulb Light E27 220V RGB+W+C" in description.
I've seen several mentions of id ZB-RGBCW on git, but without any model number.

Reply:
No problem, can you provide a picture of the device?

Reply:
Sure.
![ZB-RGBCW](https://user-images.githubusercontent.com/59430858/100967977-8a0a2880-3541-11eb-9f58-7ee884bd1a80.png)

Reply:
Thanks!